td-shim-tools/src/enroller.rs

```diff
@@ -26,6 +26,21 @@ use crate::public_key::{
 };
 use crate::{write_u24, InputData, OutputFile};
 
+fn update_checksum(data: &mut [u8]) {
+    // Clear checksum bit to zero
+    data[16] = 0x0u8;
+
+    let mut sum = 0x0u8;
+    for i in 0..24 {
+        if i == 17 || i == 23 {
+            continue;
```

Comment:
please dont hardcode any value.

Reply:
Removed all the hardcode implementation.